src/patterns/organisms/footer/footer-full.hbs

```diff
@@ -0,0 +1,53 @@
+---
+name: Footer with all options
+description: |
+  The optional elements are: logo, footer-sections (with links), language selector, and social media icons.
+notes: |
+  - It looks a little funny on the protocol page becuase the media queries expect it to be the full width of the page.
+links:
+    Footer Demo: /demos/footer.html
+---
+
+<footer class="mzp-c-footer">
+  <div class="mzp-l-content">
+    <nav class="mzp-c-footer-primary">
+      {{#embed "patterns.organisms.footer.includes.footer-primary-logo"}}{{/embed}}
+      <div class="mzp-c-footer-sections">
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+        {{#embed "patterns.organisms.footer.includes.footer-section"}}{{/embed}}
+      </div>
+    </nav>
+
+    <nav class="mzp-c-footer-secondary">
+      {{#embed "patterns.organisms.footer.includes.footer-language"}}{{/embed}}
+      {{#embed "patterns.organisms.footer.includes.footer-links-social"}}{{/embed}}
+      <div class="mzp-c-footer-legal">
+        <p class="mzp-c-footer-license">
+          Visit Mozilla Corporation’s not-for-profit parent, the <a href="https://foundation.mozilla.org/">Mozilla Foundation</a>.<br>
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Portions of this content are ©1998–2018 by individual mozilla.org contributors.
+          Content available under a <a rel="license" href="https://www.mozilla.org/foundation/licensing/website-content/">Creative Commons license</a>.
+        </p>
+        <ul class="mzp-c-footer-terms">
+          <li><a href="https://www.mozilla.org/privacy/websites/">Website Privacy Notice</a></li>
+          <li><a href="https://www.mozilla.org/privacy/websites/#cookies">Cookies</a></li>
+          <li><a href="https://www.mozilla.org/about/legal/">Legal</a></li>
+          <li><a href="https://www.mozilla.org/privacy/websites/">Website Privacy Notice</a></li>
+          <li><a href="https://www.mozilla.org/privacy/websites/#cookies">Cookies</a></li>
+          <li><a href="https://www.mozilla.org/about/legal/">Legal</a></li>
+        </ul>
+      </div>
+    </nav>
+  </div>
+</footer>
+<script src="{{@root.baseurl}}/assets/protocol/protocol/js/protocol-utils.js"></script>
+<script src="{{@root.baseurl}}/assets/protocol/protocol/js/protocol-supports.js"></script>
```

Comment:
Should `protocol-supports.js` and `protocol-utils.js` be included in the base template for the docs site? I notice the navigation now has a dependency on the feature detect library, but it's not included in the navigation demo page, so it's falling back to the CSS-only interactions.

Reply:
Added to the site template.